tooling/api/scripts/after-build.cjs

```diff
@@ -5,14 +5,11 @@
 const { readFileSync, readdirSync, writeFileSync, copyFileSync } = require('fs')
 const { copySync } = require('fs-extra')
 
-/**
- * append our api modules to `exports` in `package.json` then write it to `./dist`
- */
+// append our api modules to `exports` in `package.json` then write it to `./dist`
 const pkg = JSON.parse(readFileSync('package.json', 'utf8'))
```

Comment:
What if there is no `package.json`?

Reply:
if there is no `package.json`, the build would fail anyway and this `after-build` script is useless